scripts/globals/hunts.lua

```diff
@@ -0,0 +1,1616 @@
+        [ 554] = { huntId =  52436, optionId = 1697 },
+        [ 810] = { huntId =  69845, optionId = 1705 },
+        [1066] = { huntId =  69846, optionId = 1713 },
+        -- Rolanberry Fields
+        [ 306] = { huntId =  52439, optionId = 1721 },
+        [ 562] = { huntId =  69848, optionId = 1729 },
+        [ 818] = { huntId =  68825, optionId = 1737 },
+        [1074] = { huntId =  87258, optionId = 1745 },
+        -- Palborough Mines
+        [ 314] = { huntId =  17627, optionId = 1753 },
+        [ 570] = { huntId =  35036, optionId = 1761 },
+        [ 826] = { huntId =  34013, optionId = 1769 },
+        [1082] = { huntId =  35038, optionId = 1777 },
+        -- Dangruf Wadi
+        [ 322] = { huntId =  35039, optionId = 1785 },
+        [ 578] = { huntId =  36064, optionId = 1793 },
+        [ 834] = { huntId =  53473, optionId = 1801 },
+        -- Korroloka Tunnel
+        [ 330] = { huntId =  53474, optionId = 1809 },
+        [ 586] = { huntId =  53475, optionId = 1817 },
+        [ 842] = { huntId =  54500, optionId = 1825 },
+        [1098] = { huntId =  68837, optionId = 1833 },
+        -- Gusgen Mines
+        [ 338] = { huntId =  69862, optionId = 1841 },
+        [ 594] = { huntId =  69863, optionId = 1849 },
+        [ 850] = { huntId =  69864, optionId = 1857 },
+        [1106] = { huntId =  69865, optionId = 1865 },
+        [1362] = { huntId =  69866, optionId = 1873 },
+        [1618] = { huntId =  69867, optionId = 1881 },
+        -- Crawlers' Nest
+        [ 346] = { huntId = 104684, optionId = 1889 },
+        [ 602] = { huntId = 103661, optionId = 1897 },
+        [ 858] = { huntId = 138478, optionId = 1905 },
+        -- Beadeaux
+        [ 354] = { huntId =  35055, optionId = 1913 },
+        [ 610] = { huntId =  52464, optionId = 1921 },
+        [ 866] = { huntId =  52465, optionId = 1929 },
+        [1122] = { huntId =  69874, optionId = 1937 },
+        [1378] = { huntId =  86259, optionId = 1945 },
+        -- Oldton Movalpolos
+        [ 362] = { huntId =  86260, optionId = 1953 },
+        [ 618] = { huntId =  88309, optionId = 1961 },
+        [ 874] = { huntId =  86262, optionId = 1969 },
+        -- Newton Movalpolos
+        [ 370] = { huntId = 139511, optionId = 1977 },
+        [ 626] = { huntId = 140536, optionId = 1985 },
+        [ 882] = { huntId = 155897, optionId = 1993 },
+        -- Habitat Unknown (Bastok Mines)
+        [ 378] = { huntId = 157220, optionId = 2001 },
+        [ 634] = { huntId = 157221, optionId = 2009 },
+        [ 890] = { huntId = 174627, optionId = 2017 },
+
+      hunt =
+      {
+        [1577] = { pageId = 197, mobId = 17211561, bounty =  5, fee =   0 }, -- Stinging Sophie
+        [1585] = { pageId = 198, mobId = 17211714, bounty =  5, fee =   0 }, -- Maighdean Uaine
+        [1593] = { pageId = 199, mobId = 17211666, bounty =  5, fee =   0 }, -- Bedrock Barry
+        [1601] = { pageId = 200, mobId = 17215888, bounty =  5, fee =   0 }, -- Leaping Lizzy
```

Comment:
I'm not sure if this is very common, but I happened to test this on Lizzy and she actually has 2 IDs, so might need to add the other one `17215868`.

Reply:
I asked about this and got no reply unfortunately. There are some NMs that have a single spawn point under one ID, and like 50 spawn points under another ID. In those cases, I opted for the 50. I don't know what the logic is there exactly.

Reply:
tabling all the IDs of all these NMs is the wrong approach. Followup what all existing regimes do and include the regime ID at the mob, then match on regime ID instead of performing a lookup to see if that specific mob is in a table. We don't need to keep track of them all because its never going to change, and you already have that line there anyway. model it after the existing regimes instead of trying to centralize every bit of data to one place as this is a case where that just gets in the way.

`tpz.hunts.checkHunt(mob, player, isKiller)`
compare with
`tpz.regime.checkRegime(player, mob, 80, 1, tpz.regime.type.FIELDS)`
and
`tpz.regime.checkRegime(player, mob, 640, 2, tpz.regime.type.GROUNDS)`
you're trying to reinvent the wheel to separate hunts from these (understandable, a lot of code doesn't apply here) but missed why the regime ID was included.

now looking at this:
`tpz.hunts.checkHunt(mob, player, isKiller)`
We should not need isKiller at all as in retail all member who have the hunt regime active get credit and its not going to stack up (onMobDeath code will run once-per-member), so lets just go with:
`tpz.hunts.checkHunt(mob, player, RegimeIDhere)` to minimize your changes.

Reply:
p.s. if you're thinking "but then we need to keep track of which NMs go in which regime!" no, because the regime IDs will never shift. set once done for life. :)

Reply:
I gotta disagree here on part of that. isKiller isn't necessary, you're right, but frankly I think having all the mobs tabled here with a simple lookup to a table is a much faster and better method than breaking out every ID into separate mob scripts. Regimes takes the ID it gets and combs through a table using a for loop anyway to find the mob which is far from elegant. This solution involves one direct check to one table and I gotta go with the performance method here.

Reply:
I think this works well, and if you believe you have a better solution I welcome you to PR the files after this merges. :)

Reply:
yeah no, I'm no opening a competing PR just because you prefer listing every NMs ID into the table and then trying to make workaround for the inherent shortcomings of that approach. 

A given NM can have multiple IDs. at a minimum would would need multiple ID entries. There is no reason we could not match on regime ID instead of mobID, loop or no loop.

Reply:
Switching to `tpz.hunts.checkHunt(mob, player, RegimeIDhere)` would actually solve the multiple ID mobs issue all at once and would require minimal changes. `checkHunt` becomes:
```lua
    function tpz.hunts.checkHunt(mob, player, RegimeIDhere)
        local huntZone = player:getCharVar("[hunt]zone")
        local huntId = player:getCharVar("[hunt]id")
        local playerHunt = zone[huntZone].hunt[huntId]

        -- required NM has been defeated
        if player:getCharVar("[hunt]status") == 1 and huntId == RegimeIDhere then
           player:messageBasic(tpz.msg.basic.FOV_DEFEATED_TARGET + 20)
           player:setCharVar("[hunt]status", 2)
        end
    end
```
but that would also require removing all the mob IDs from the tables and updating all the mob files as well.

Reply:
@cocosolos that is what I was going for. It does suck to re-edit all those mob scripts, but they all did have to have an edit to start with so its not like these didn't have code going to them. That said I will volunteer to make the mob script changes if apoc wants to take this road.

the alternative is harder - alterations to support multiple mobID elements and then the inevitable future problems that come with mobIDs later